### ISSUE TITLE
Missing Favorites plugin

### ISSUE THREAD

**Issue by robdecker** [NotePlan/plugins#474]
**Describe the bug**
The ⭐️ Favorites plugin is not available to install

**To Reproduce**
1. Go to 'Settings'
2. Click on 'AI & Plugins'
3. Wait for the list of plugins to fully populate
4. Look for '⭐️ Favorites'

**Expected behavior**
The plugin should be visible and installable.

**Screenshots**
![2023-08-10 at 12 40 31](https://github.com/NotePlan/plugins/assets/852280/4bedf637-e05a-4d5b-b06c-6de44e9cb978)

**System**
 - Device: Mac
 - OS: macOS 13.4.1
 - NotePlan Version: 3.9.5
 - Plugin Name & Version: ⭐️ Favorites

**Comment by dwertheimer** [comment]
@robdecker Thank you for reporting this! That's so weird. Somewhere along the way, it got deleted. I wonder how long it's been gone! I will re-release right now. Please let me know if you're able to find it. 

I'm curious, how did you know about the plugin?

**Comment by robdecker** [comment]
I see it now, thanks!

It's listed on the [plugins list](https://github.com/NotePlan/plugins/blob/main/Plugin-Listing.md).

**Comment by dwertheimer** [comment]
@robdecker Oh my. I forgot that list exists. I made that script so long ago and people stopped updating it. The list hasn't been updated in a year! Thanks for letting me know.

BTW, I created a new, different version of this inside of NotePlan. Download this plugin and run the 2nd command listed below:
<img width="633" alt="Screen Cap 2023-08-10 at 12 58 56@2x" src="https://github.com/NotePlan/plugins/assets/8949588/1a45d4ae-710a-4821-9b27-2b6f3624dfe3">
